eng/Versions.props

```diff
@@ -1,17 +1,18 @@
 <Project>
   <PropertyGroup>
     <!-- The .NET product branding version -->
-    <ProductVersion>7.0.0</ProductVersion>
+    <ProductVersion>8.0.0</ProductVersion>
     <!-- File version numbers -->
-    <MajorVersion>7</MajorVersion>
+    <MajorVersion>8</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
     <SdkBandVersion>7.0.100</SdkBandVersion>
-    <PreReleaseVersionLabel>rc</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>alpha</PreReleaseVersionLabel>
     <PreReleaseVersionIteration>1</PreReleaseVersionIteration>
+    <PreviousReleaseMajorVersion>7</PreviousReleaseMajorVersion>
     <!-- Set assembly version to align with major and minor version,
          as for the patches and revisions should be manually updated per assembly if it is serviced. -->
-    <AssemblyVersion>$(MajorVersion).$(MinorVersion).0.0</AssemblyVersion>
+    <AssemblyVersion>$(PreviousReleaseMajorVersion).$(MinorVersion).0.0</AssemblyVersion>
```

Comment:
```suggestion
    <AssemblyVersion>7.$(MinorVersion).0.0</AssemblyVersion>
```

Reply:
I'll apply your suggestion, but can you please explain why this value can't stay in `$(MajorVersion)`? Why do we need to use 7 instead of 8?

Reply:
In the past release when we branded main to .NET 7 we didn't bump the assembly versions as part of the branding update. We only incremented the package major version to avoid bin clashes when packages being uploaded to the same feed.

Incrementing the assembly versions should happen afterwards to not block this PR: I assume that the assembly update could cause other issues that would need to be resolved.